apps/src/applab/Exporter.js

```diff
@@ -1,15 +1,14 @@
 /* global dashboard */
 import $ from 'jquery';
 
-var JSZip = require('jszip');
-var saveAs = require('filesaver.js').saveAs;
-
-var assetPrefix = require('../assetManagement/assetPrefix');
-var download = require('../assetManagement/download');
-var elementLibrary = require('./designElements/library');
-var exportProjectEjs = require('../templates/exportProject.html.ejs');
-var exportProjectReadmeEjs = require('../templates/exportProjectReadme.md.ejs');
-var logToCloud = require('../logToCloud');
+import JSZip from 'jszip';
+import {saveAs} from 'filesaver.js';
+import assetPrefix from '../assetManagement/assetPrefix';
+import download from '../assetManagement/download';
+import elementLibrary from './designElements/library';
```

Comment:
Likewise some of these.  assetPrefix.js also has multiple exports old-style.  I remember this causing some issues before we converted utils.js to export ES6-style.